tests/nosetests/pyanaconda_tests/module_payload_shared.py

```diff
@@ -152,6 +151,6 @@ def check_sources(self, expected_sources=None):
         self._test.assertEqual(self.payload_interface.Sources, expected_paths)
 
         if expected_sources:
-            self._test.assertTrue(self.payload_interface.HasSource())
+            self._test.assertGreater(len(self.payload_interface.Sources), 0)
```

Comment:
I think this is the same as `self._test.assertTrue(self.payload_interface.Sources)`.

Reply:
I had something similar there, but didn't like the implicit (unstated) conversion to `bool`. Actually, how about this:
```
self._test.assertEqual(
  bool(expected_sources),
  self.payload_interface.Sources == []
)
```

Reply:
Actually, you can remove this block, because the property Sources is tested above.